README.md

```diff
@@ -77,7 +98,31 @@ With any push to `main`, in order to deploy changes, the `Terraform Cloud` varia
 - ENVIRONMENT: Option can be `development` or `production`. 
 This effects the default values on the NWP and Satellite pages
 
-## Contributors 
+## Develop
+
+Currently this repository is only used by OCF for internal metric calculations, as it contiains client information. We hope to make it more freely useable in the near future.
+
+### Tests
+
+To run the tests, make sure you have `pytest` installed
+```bash
+pip install pytest
+```
+and then you can run
+```bash
+pytest
+```
+
+## Contributors and community
+
+
+[![issues badge](https://img.shields.io/github/issues/openclimatefix/elexonpy?color=FFAC5F)](https://github.com/openclimatefix/elexonpy/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)
+
+- PR's are welcome! See the [Organisation Profile](https://github.com/openclimatefix) for details on contributing
+- Find out about our other projects in the [OCF Meta Repo](https://github.com/openclimatefix/ocf-meta-repo)
+- Check out the [OCF blog](https://openclimatefix.org/blog) for updates
+- Follow OCF on [LinkedIn](https://uk.linkedin.com/company/open-climate-fix)
+- OCF templete: (https://github.com/openclimatefix/ocf-template?tab=readme-ov-file#contributing-and-community)
 
```

Comment:
at the very bottom can you add 

![Screenshot 2024-11-15 at 09 30 58](https://github.com/user-attachments/assets/e68402cb-927a-4df3-8e07-a4c7e17b839e)